rkt/gc.go

```diff
@@ -111,7 +111,7 @@ func emptyExitedGarbage(gracePeriod time.Duration) error {
 			if err := p.ExclusiveLock(); err != nil {
 				return
 			}
-			stdout("Garbage collecting pod %q", p.uuid)
+			stderr("Garbage collecting pod %q", p.uuid)
```

Comment:
At some point we probably want to print this information in a machine-readable form